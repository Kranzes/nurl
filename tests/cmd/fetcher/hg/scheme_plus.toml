args = ["hg+https://hg.sr.ht/~scoopta/wofi", "v1.3", "--fetcher", "fetchhg"]
