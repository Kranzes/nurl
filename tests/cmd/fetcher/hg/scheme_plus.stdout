fetchhg {
  url = "https://hg.sr.ht/~scoopta/wofi";
  rev = "v1.3";
  sha256 = "sha256-GxMjEXBPQniD+Yc9QZjd8TH4ILJAX5dNzrjxDawhy8w=";
}